pytorch_lightning/logging/comet_logger.py

```diff
@@ -1,25 +1,78 @@
+from torch import is_tensor
+
 try:
     from comet_ml import Experiment as CometExperiment
+    from comet_ml.papi import API
 except ImportError:
     raise ImportError('Missing comet_ml package.')
 
 from .base import LightningLoggerBase, rank_zero_only
 
 
 class CometLogger(LightningLoggerBase):
-    def __init__(self, *args, **kwargs):
+    def __init__(self, api_key, workspace, rest_api_key=None, project_name=None, experiment_name=None, *args, **kwargs):
+        """
+        Initialize a Comet.ml logger
+
+        :param api_key: API key, found on Comet.ml
+        :param workspace: Name of workspace for this user
+        :param project_name: Optional. Send your experiment to a specific project.
+        Otherwise will be sent to Uncategorized Experiments.
+        If project name does not already exists Comet.ml will create a new project.
+        :param rest_api_key: Optional. Rest API key found in Comet.ml settings. This is used to determine version number
+        :param experiment_name: Optional. String representing the name for this particular experiment on Comet.ml
+        """
         super(CometLogger, self).__init__()
-        self.experiment = CometExperiment(*args, **kwargs)
+        self.experiment = CometExperiment(api_key=api_key, workspace=workspace, project_name=project_name, *args,
```

Comment:
You didn't introduce this problem, but maybe you can fix it while you're working on this class. We need to create the experiment lazily so that we don't get duplicates when running in distributed mode. If you're willing to take a stab at fixing this, take a look at one of the other loggers to see how this is done.

Reply:
Hi Nic, thanks for the tips. I've just pushed a version of the code where I attempt to create the experiment lazily, but it's a bit outside my comfort zone at the moment so please let me know if there's a way I could improve the code.

Additionally, I've noticed that the Travis-ci testing has failed the same way for all PRs in the last 24 hours or so. Is that something that can be addressed easily?